roles/easy-build/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+eb_tmp_dir: /tmp/easybuild
+eb_bootstrap_url: 'https://raw.githubusercontent.com/easybuilders/easybuild-framework/develop/easybuild/scripts/bootstrap_eb.py'
```

Comment:
Should we peg this to a specific commit hash rather than just the develop branch?

Reply:
@issamsaid , can you change this to : https://raw.githubusercontent.com/easybuilders/easybuild-framework/develop/easybuild/scripts/bootstrap_eb.py

Reply:
Yes.